src/styles/ErrorPage.css

```diff
@@ -0,0 +1,26 @@
+* {
+   margin: 0;
+   padding: 0;
+}
+
+.errorPage {
+    position: absolute;
```

Comment:
@Aaishpra Can I ask why are you using absolute positioning when this can be achieved with relative positioning as well. This way the overlapping issue on small screens mentioned by @codesankalp will also resolve.

Reply:
` relative` is not gonna work on larger screen